pkg/asset/manifests/aws/zones.go

```diff
@@ -263,6 +345,7 @@ func extractZonesFromInstallConfig(in *zonesInput) (*zonesCAPI, error) {
 		// Ignoring as edge pool is not yet supported by CAPA.
 		// See https://github.com/openshift/installer/pull/8173
 		if pool.Name == types.MachinePoolEdgeRoleName {
+			out.edgeZones.Insert(pool.Platform.AWS.Zones...)
 			continue
```

Comment:
Do you still want this continue here?

Reply:
Ah, if it's an edge pool you don't want to `SetDefaultConfigZones`.

Reply:
that's correct, edge zones arent eligible for regular zones

Reply:
edge zones isn't eligible to set default zones (next step), so continue here is correct. I believe it is missing a check for some unexpected situation when there is no zones defined in the edge pool (or should we skip this considering the [validation is performing](https://github.com/openshift/installer/blob/1c9f90d42d2869a6fe95848b9e1c8ee9cda401d4/pkg/asset/installconfig/aws/validation.go#L264-L276) those checks?).

I added it in my recent changes just to make sure a coverage in the tests.